docs/data/material/components/text-fields/text-fields.md

```diff
@@ -44,8 +44,8 @@ The `helperText` prop can then be used to provide feedback to the user about the
 
 ## Multiline
 
-The `multiline` prop transforms the text field into a [TextareaAutosize](/material-ui/react-textarea-autosize/) element.
-Unless the `rows` prop is set, the height of the text field dynamically matches its content (using [TextareaAutosize](/material-ui/react-textarea-autosize/)).
+The `multiline` prop transforms the Text Field into a [Base UI Textarea Autosize](/base-ui/react-textarea-autosize/) element.
```

Comment:
off-topic: I find this convention weird. As a user I always prefer to read the actual component name:

- "TextField" vs "Text Field"
- "TextareaAutosize" vs "Textarea Autosize"
- "ClickAwayListener" vs "Click-Away Listener"

Reply:
Interesting. This was a style convention we adopted a few years ago to try to help improve readability as I recall. But if it's not actually helping then maybe we should consider re-evaluating it as an org.